### PR TITLE
[INJICERT-189] add changes to install mosip id plugin from artifactory

### PR DESCRIPTION
**Pull request by challabeehyv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Execute installation script
      * For certify:
        * By default, the plugin will be taken from artifactory server
        * For custom plugin:
-         * In the [docker compose file](docker-compose/docker-compose-certify/docker-compose.yml) uncomment the [enable_certify_artifactory](docker-compose/docker-compose-certify/docker-compose.yml#L74) and [volume](docker-compose/docker-compose-certify/docker-compose.yml#L78)
+         * In the [docker compose file](docker-compose/docker-compose-certify/docker-compose.yml) uncomment the [enable_sunbird_certify_artifactory](docker-compose/docker-compose-certify/docker-compose.yml#L74) and [volume](docker-compose/docker-compose-certify/docker-compose.yml#L78)
```
Comment (vishwa-vyom) [inline README.md:57]: why is this called enable_sunbird_certify_artifactory ? the artifactory is not specific to sunbir

File: certify-service/Dockerfile [hunk 2]
```diff
@@ -45,8 +45,14 @@ ENV is_glowroot_env=${is_glowroot}
...
-# environment variable for certify artifactory plugins to pass at docker run time
-ENV enable_certify_artifactory="true"
+# environment variable for sunbird certify artifactory plugins to pass at docker run time
+ENV enable_sunbird_certify_artifactory="true"
+
+# environment variable for mosip identity certify artifactory plugins to pass at docker run time
+ENV enable_mosipid_certify_artifactory="false"
+
+# environment variable for mock identity certify artifactory plugins to pass at docker run time
+ENV enable_mockid_certify_artifactory="false"
```
Comment (vishwa-vyom) [inline certify-service/Dockerfile:55]: what will happen tomorrow a new plugin is created by a external team ? 
We cannot have any reference to mosipid/sunbird/mock in the certify code.

Comment (vishwa-vyom) [reply certify-service/Dockerfile:55]: this decision should be based on the config "mosip.certify.integration.vci-plugin"

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-service/Dockerfile (hunk 2 above)
Comment (challabeehyv) [reply certify-service/Dockerfile]: extracted all the plugins with single variable, so the implementation will be considered based on config properties

File: README.md (hunk 1 above)
Comment (challabeehyv) [reply README.md:57]: changed to enable_certify_artifactory

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**